sky/server/common.py

```diff
@@ -211,7 +218,10 @@ def _start_api_server(deploy: bool = False,
         # If this is called from a CLI invocation, we need
         # start_new_session=True so that SIGINT on the CLI will not also kill
         # the API server.
-        subprocess.Popen(cmd, shell=True, start_new_session=True)
+        subprocess.Popen(cmd,
+                         shell=True,
+                         start_new_session=True,
+                         cwd=working_dir)
```

Comment:
Ahh, this could be a bit dangerous, if a user specified a relative path for `SKYPILOT_CONFIG=./config.yaml`. Does this mean the config path will not be loaded locally?

Reply:
We could use [python `-P` CLI option](https://docs.python.org/3/using/cmdline.html#cmdoption-P), but this is only available in 3.11 and higher.

Reply:
Great catch! Will investigate more